src/components/transactions/TxFilterForm/index.tsx

```diff
@@ -0,0 +1,223 @@
+              <FormControl>
+                <FormLabel>Transaction type</FormLabel>
+                <Controller
+                  name={FILTER_TYPE_FIELD_NAME}
+                  control={control}
+                  render={({ field }) => (
+                    <RadioGroup {...field}>
+                      {Object.values(TxFilterType).map((value) => (
+                        <FormControlLabel value={value} control={<Radio />} label={value} key={value} />
+                      ))}
+                    </RadioGroup>
+                  )}
+                />
+              </FormControl>
+            </Grid>
+            <Grid item xs={12} md={9}>
+              <FormControl sx={{ width: '100%' }}>
+                <FormLabel sx={{ mb: 1 }}>Parameters</FormLabel>
+                <Grid container spacing={2} xs={12}>
+                  {!isModuleFilter && (
+                    <>
+                      <Grid item xs={12} md={6}>
+                        <DatePickerInput name={DATE_FROM_FIELD_NAME} label="From" />
+                      </Grid>
+                      <Grid item xs={12} md={6}>
+                        <DatePickerInput name={DATE_FROM_FIELD_NAME} label="To" />
+                      </Grid>
+                      <Grid item xs={12} md={6}>
+                        <Controller
+                          name={AMOUNT_FIELD_NAME}
+                          control={control}
+                          rules={{ validate: validateAmount }}
+                          render={({ field, fieldState }) => (
+                            <TextField
+                              label="Amount"
+                              helperText={fieldState.error?.message}
+                              error={!!fieldState.error}
+                              {...field}
+                              fullWidth
+                            />
+                          )}
+                        />
+                      </Grid>
+                    </>
+                  )}
+                  {isIncomingFilter && (
+                    <Grid item xs={12} md={6}>
+                      <AddressBookInput label="Token address" {...register(TOKEN_ADDRESS_FIELD_NAME)} fullWidth />
+                    </Grid>
+                  )}
+                  {isMultisigFilter && (
+                    <>
+                      <Grid item xs={12} md={6}>
+                        <AddressBookInput label="Recipient" {...register(RECIPIENT_FIELD_NAME)} fullWidth />
+                      </Grid>
+                      <Grid item xs={12} md={6}>
+                        <Controller
+                          name={NONCE_FIELD_NAME}
+                          control={control}
+                          rules={{ validate: validateAmount }}
+                          render={({ field, fieldState }) => (
+                            <TextField
+                              label="Nonce"
+                              helperText={fieldState.error?.message}
```

Comment:
In other places we display the error message in the `label` as to not cause any layout shift. Wdyt?

Reply:
I'll adjust it to keep it uniform.